webserv.json

```diff
@@ -733,15 +733,15 @@
             "method": "POST",
             "sortNum": 220000,
             "created": "2024-02-22T04:30:25.199Z",
-            "modified": "2024-02-22T05:08:40.718Z",
+            "modified": "2024-02-22T13:53:54.289Z",
             "headers": [
                 {
                     "name": "Content-Length",
                     "value": "73"
                 },
                 {
                     "name": "Content-Type",
-                    "value": "image/jpeg"
```

Comment:
is this on purpose changed from jped to image?

Reply:
we are uploading a text file there, so I wanted to change the content type to match